src/rbac/config/db/database.py

```
@@ -15,10 +16,20 @@
                        pool_size=10, max_overflow=0,
                        pool_recycle=300)
+SessionLocal = sessionmaker(bind=engine, autocommit=False, autoflush=False)
 
+def get_db():
+    db: Session = SessionLocal()
+    try:
+        yield db
+    except SQLAlchemyError as e:
+        logger.error(f"Database connection error: {e}")
+        raise HTTPException(status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+                            detail="요청하신 서비스가 잠시 내부적으로 문제가 발생하였습니다.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)